modules/claims_api/spec/requests/v1/disability_compensation_request_spec.rb

```diff
@@ -239,6 +239,26 @@
           ]
         end
 
+        context "when 'treatments[].center.name' is an empty string'" do
```

Comment:
Could also add a check here for making sure it behaves properly when the value is > 50 characters.

Reply:
good idea, adding that in, as well